modules/claims_api/app/controllers/concerns/claims_api/endpoint_deprecation.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module EndpointDeprecation
+    extend ActiveSupport::Concern
+
+    included do
+      V0_DEV_DOCS = 'https://developer.va.gov/explore/benefits/docs/claims?version=0.0.1'
+      V1_DEV_DOCS = 'https://developer.va.gov/explore/benefits/docs/claims?version=current'
```

Comment:
might be worth putting these in Settings instead?

Reply:
Do you mean storing them [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/settings.yml#L414)?

Reply:
yes